agent-control/src/sub_agent/on_host/supervisor.rs

```diff
@@ -433,14 +433,18 @@ pub mod tests {
 
     const DURATION_DELTA: std::time::Duration = Duration::from_millis(100);
 
-    #[cfg(target_family = "unix")] //TODO This should be removed when Windows support is added
     #[traced_test]
     #[rstest]
-    #[case::long_running_process_shutdown_after_start(
+    #[cfg_attr(target_family = "unix",case::long_running_process_shutdown_after_start(
         "long-running",
         ExecutableData::new("sleep".to_owned(), "sleep".to_owned()).with_args(vec!["10".to_owned()]),
         Some(Duration::from_secs(1)),
-        vec!["Stopping executable", "Executable terminated"])]
+        vec!["Stopping executable", "Executable terminated"]))]
+    #[cfg_attr(target_family = "windows",case::long_running_process_shutdown_after_start(
+        "long-running",
+        ExecutableData::new("cmd".to_owned(), "cmd".to_owned()).with_args(vec!["/C".to_owned(), "timeout".to_owned(), "/T".to_owned(), "10".to_owned(), "/NOBREAK".to_owned()]),
+        Some(Duration::from_secs(1)),
+        vec!["Stopping executable", "Executable terminated"]))]
```

Comment:
I guess the `target_family = "unix"` was added because the use case doesn't work yet. If that wasn't the reason why adding it, I'd also remove the windows part.

Reply:
I totally misread it! Thanks for the explanation!